### ISSUE TITLE
Cant run solid init

### ISSUE THREAD

**Issue by melvincarvalho** [solid/node-solid-server#465]
Running a fresh install node v6.1.0 on ubuntu linux 16.04

The following error is thrown after completing the config questions

```
/node_modules/rx/dist/rx.js:77
    throw e;
    ^
Nothing found at this path
```

**Comment by dduran1967** [comment]
I've seen this error when attempting to use the default value for path to the data viewer app.  I was able to complete the init process by entering a valid directory.  If that doesn't work, maybe you could post the values you are using.

**Comment by melvincarvalho** [comment]
@dduran1967 thanks for the additional info.  I dont use `solid init` anymore, instead I use a pre existing config file which I have saved.

```
{
  "root": "<root>",
  "port": "8000",
  "webid": true,
  "auth": "tls",
  "sslKey": "<key>",
  "sslCert": "<cert>",
  "idp": false,
  "fileBrowser": "https://linkeddata.github.io/warp/#/list/",
  "dataBrowser": true,
  "strictOrigin": false
}
```
What I sometimes add is "dataBrowserPath" : "default"

**Comment by dduran1967** [comment]
"default" doesn't work for me either but "/" does.

**Comment by melvincarvalho** [comment]
I added the priority tag to this issue, as I believe that it blocks fresh installs of node solid server from working.

Please, let me know if there is some better tag or workflow that can be used.

**Comment by ryankeener** [comment]
Looks like there are a couple things going on here. 

I see that @dmitrizagidulin has addressed this particular issue in 9e75613 but I noticed that Inquirer [wasn't handling Promise rejections correctly](https://github.com/SBoudrias/Inquirer.js/issues/363) I've submitted [the following PR](https://github.com/SBoudrias/Inquirer.js/pull/471) which addresses that issue.

**Comment by dmitrizagidulin** [comment]
@ryankeener thanks for the heads up!

**Comment by dmitrizagidulin** [comment]
Should be fixed in 3.4.1 / latest master. Closing.
